packages/ui/icons/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from '@littlespoon/icons'
```

Comment:
Can you also create a file `ArrowIcon.tsx` and export it there? This will make it easier to do:

```tsx
import ArrowIcon from '@littlespoon/ui/ArrowIcon'
```